nodejs/week4/warmup/index.js

```diff
@@ -0,0 +1,80 @@
+class Email {
+  constructor(subject, body) {
+    this.subject = subject;
+    this.body = body;
+  }
+}
+
+class SpamDetector {
+  upperCaseCheck(emailBody) {
+    const string = emailBody.body;
+    const stringSplit = string.split("");
+    const upperCaseArray = stringSplit.filter(item => {
+      return item == item.toUpperCase();
+    });
+
+    const percentage = (upperCaseArray.length * 100) / stringSplit.length;
+
+    if (percentage > 60) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  badWordCheck(emailBody) {
+    const strings = emailBody.body;
+
+    const stringsSplit = strings.split(" ");
+
+    const badWords = ["viagra", "offer", "free", "business proposal"];
+
+    const badWordsInEmail = stringsSplit.filter(item => {
+        
+      return badWords.includes(item.toLowerCase());
+
+    });
+
+    if (badWordsInEmail.length !== 0) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  subjectCheck(subject) {
+    const emailSubject = subject.subject;
+
+    if (emailSubject.toLowerCase() === "hello") {
```

Comment:
You can simplify all of your return statements in the way I described previously 🙂